_includes/medium_posts.html

```diff
@@ -15,7 +14,7 @@ <h3 class="team-item-title">
         <a href="{{ cover_post.medium_url }}" target="_blank" style="text-decoration: none;">{{ cover_post.medium_title }}</a>
       </h3>
       <img src="https://cdn-images-1.medium.com/fit/t/800/240/{{ cover_post.medium_preview_image_id }}" class="img-responsive" style="margin: 2px;width:100%;">
-      <p class="team-item-subtitle">{{ cover_post.medium_subtitle }}</p>
+      <p class="team-item-subtitle">{{ cover_post.medium_subtitle }} <small>– {{cover_post.medium_published_at | divided_by: 1000 | date: '%d/%m/%Y'}}</small></p>
```

Comment:
Il formato delle date va localizzato in base alla lingua

Reply:
done  73fc1cb64dee468580e85e9401f570a981e390da